go/cmd/dolt/commands/cnfcmds/auto_resolve.go

```diff
@@ -135,6 +135,12 @@ func ResolveTable(ctx context.Context, dEnv *env.DoltEnv, root *doltdb.RootValue
 		return err
 	}
 
+	// Resolving conflicts for one table, will not resolve conflicts on another.
+	err = execute(sqlCtx, eng, "SET DOLT_ALLOW_COMMIT_CONFLICTS = 1;")
```

Comment:
I'm concerned that setting this variable in library could lead to unintended consequences. Is this code path exposed through a SQL function or store procedure?

Reply:
It isn't used in any other functions, but good call regardless. Added. :)